solution-troubleshooting-init-error-status-error-message-after-deploying-ondat.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Solution - Troubleshooting 'Init:Error' Status Error Message After Deploying Ondat"
+linkTitle: "Solution - Troubleshooting 'Init:Error' Status Error Message After Deploying Ondat"
+---
+
+## Issue
+
+When attempting to deploy Ondat into an OpenShift or Kubernetes cluster, you notice that the Ondat daemonset set pods are stuck in a `Init:Err` state. Below is an example of the error message being reported under the `STATUS` column.
+
+```bash
+# Get the status of the pods in the "storageos" namespace.
+kubectl get pods --namespace storageos
+
+NAME                                                 READY   STATUS    RESTARTS   AGE
+# Truncated output...
+storageos-node-8fhf6                                 0/3     Init:Err  0          6s
+storageos-node-8z77g                                 0/3     Init:Err  0          6s
+storageos-node-pzvp7                                 0/3     Init:Err  0          6s
+storageos-node-qbjbr                                 0/3     Init:Err  0          6s
+storageos-node-vkj92                                 0/3     Init:Err  0          6s
+# Truncated output...
+```
+
+## Root Cause
+
+The root cause of this issue is due to missing [Linux-IO (LIO) related kernel modules](https://en.wikipedia.org/wiki/LIO_%28SCSI_target%29) on worker nodes that are required for Ondat to successfully start up and run.
+
+- The Ondat daemonset will attempt to load the required kernel modules onto the worker nodes. If Ondat is unsuccessful in loading the kernel modules, an `Init:Err` error will be returned and fail Ondat from starting up without the required kernel modules.
```

Comment:
suggestion:

...and **stop** Ondat from starting up...

Reply:
fixed